src/openrct2/ride/TrackDesign.h

```diff
@@ -21,6 +21,16 @@ struct Ride;
 
 #define TRACK_PREVIEW_IMAGE_SIZE (370 * 217)
 
+/* Track Entrance entry size: 0x06 */
+struct TrackDesignEntranceElement
+{
+    int8_t z;          // 0x00
+    uint8_t direction; // 0x01
+    int16_t x;         // 0x02
+    int16_t y;         // 0x04
+};
+assert_struct_size(TrackDesignEntranceElement, 0x06);
```

Comment:
As this is no longer read from the file directly the struct size isn't important. So the assert can be deleted. This will allow us in the future to add or adjust the fields.